react-native/OpeningaPDF/ios/OpeningaPDF/AppDelegate.mm

```diff
@@ -0,0 +1,133 @@
+#import "AppDelegate.h"
```

Comment:
Should this file be AppDelegate.mm ?

Reply:
Correct. Don't know how that slipped through. It's fixed now.